Lib/tkinter/__init__.py

```diff
@@ -3130,8 +3133,8 @@ def __init__(self, master=None, cnf={}, **kw):
         STANDARD OPTIONS
 
             activebackground, activeforeground, anchor,
-            background, bitmap, borderwidth, cursor,
-            disabledforeground, font, foreground,
+            background, bitmap, borderwidth, compound,
```

Comment:
tk docs include bg, even though they do not for many of widgets above.
```suggestion
            background, bg, bitmap, borderwidth, compound,
```